slate-video-library/src/Pages/Homepage/Homepage.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Footer from "../../Components/Footer/Footer";
+import Header from "../../Components/Header/Header";
+import Sidebar from "../../Components/Sidebar/Sidebar";
+import Videocard from "../../Components/Video-Card/Videocard";
```

Comment:
You can create a merge file for imports/exports and import every component from that file.
i.e.
`import  {Footer,Header,Sidebar ,VideoCard} from "../../Components"`